man/meson.build

```diff
@@ -223,7 +223,7 @@ if git.found()
                 output : 'update-man-rules',
                 command : ['sh', '-c',
                            'cd @0@ && '.format(meson.build_root()) +
-                           'python3 @0@/tools/update-man-rules.py $(git ls-files ":/man/*.xml") >t && '.format(project_source_root) +
+                           'python3 @0@/tools/update-man-rules.py $(find @0@ -wholename "*/man/*.xml") >t && '.format(project_source_root) +
                            'mv t @0@/rules/meson.build'.format(meson.current_source_dir())],
```

Comment:
I'm not 100% convinced that this is better, but OK. I consider the fact that `git add -N man/foo.xml` is needed first to be a bonus. It allows me to write a few pages and add them in separate commits. But I can just as well do things in the opposite order, and update the rules first, for all new pages, and then use `git add -p` to select the rules updates I want. Using `find` is less "clean" than `git ls-files`, but this is a developer command, so it can be a bit shoddy, the caller is supposed to verify the resulting diff anyway. The fact that this fixes out-of-tree invocations is more important than the small downsides.

But please drop the `if git.found()` conditional, it's not useful any more.